tools/bin/acceptance_test_kube_helm.sh

```diff
@@ -86,12 +85,19 @@ if [ -n "$CI" ]; then
   echo "Printing docker disk usage after pruning..."
   docker system df
 fi
+echo "Check if server connection still alive..."
+
+if curl -sSf -o /dev/null http://localhost:8001/api/v1/health; then
+  echo "Connection is alive..."
+else
+  echo "Connection is not alive..."
+  echo "Port forwarding pod..."
+  kubectl port-forward svc/airbyte-airbyte-server-svc 8001:8001 &
+fi
```

Comment:
why do we need this port forwarding step again here?
Did you see cases where the port forwarding just stopped at this point in the script?

Reply:
Yep, after worker tests connection just dies, cause server pod dies(still uncertain why, it only happens now on Kind setup).